src/Data/Lens/Traversal.purs

```diff
@@ -108,40 +106,41 @@ failover t f s = case unwrap (t $ Star $ Tuple (Disj true) <<< f) s of
 -- | The resulting traversal is called an *affine traversal*, which
 -- | means that the traversal focuses on one or zero (if the index is out of range)
 -- | results.
-element
-  :: forall p s t a
-   . Wander p
-  => Int
-  -> Traversal s t a a
-  -> Optic p s t a a
+element ::
+  forall p s t a.
+  Wander p =>
+  Int ->
+  Traversal s t a a ->
+  Optic p s t a a
```

Comment:
Would you mind preserving the existing formatting? That'll just help this PR stay easy to review and if we switch to arrow-last style we can apply it in one commit across the repository.

Reply:
Yup, is there a vs code setting that can bring that formatting in? Apologies for the clutter!

Reply:
I changed the formatting back now :+1:
I also marked it ready for review.

Reply:
We haven’t formally adopted a formatter, so in these libraries it’s mostly about following the existing formatting until we do adopt one. Thanks!